config/default/kustomization.yaml

```diff
@@ -1,16 +1,15 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-# Adds namespace to all resources. Keep it in capm3-system, as it is a
-# dependency for CAPM3
-namespace: capm3-system
+# Adds namespace to all resources.
+namespace: ipam-system
 
 namePrefix: ipam-
 
 labels:
 - includeSelectors: true
   pairs:
-    cluster.x-k8s.io/provider: infrastructure-metal3
+    cluster.x-k8s.io/provider: ipam-metal3ipam
```

Comment:
Is it possible to have `ipam-metal3` instead? Doubling the `ipam` seems strange to me.